src/svc-hooks/eventTypes.ts

```diff
@@ -0,0 +1,24 @@
+export const validEventTypes = [
+  "build.complete",
+  // "ms.vss-release.release-abandoned-event",
```

Comment:
How'd you get this list? Maybe rather than leaving these commented out, we just include a comment w/ the command to get them again?

Reply:
https://docs.microsoft.com/en-us/azure/devops/service-hooks/events?view=azure-devops

Are you sure? It will require you to curl the page and regex it. How about I leave it commented out for now and also include the link as a comment?